### PR TITLE
Add link to command completion index

### LINKED ISSUE #1118: Add link to command completion index in getting-started.md file
[kind/Documentation]

## What did you find missing in the documentation?
Like a major feature is is not hyperlinked as other functionality

## What is the relevence of it?
I believe command completion for odo is a great feature and this should be highlighted in getting-started.md file like other major feature. For example

```
Your first application
1. Running OpenShift
2. Create an application
3. Create a component
4. Accessing the component
5. Pushing new changes to the component
6. Adding storage to the component
Command completion
* Command completion

[…]
```
It helps the user to find command completion feature and easily navigate down just by clicking it.

### PR DESCRIPTION
**Pull request by amitkrout**
## What is the purpose of this change? What does it change?

Improve getting started doc experience

## Was the change discussed in an issue?
fixes #1118

### CODE REVIEW

**Comment by amitkrout** [comment]
Also I did one cosmetic fix of ```:``` which was unnecessarily appended to index like ```Extra documentation:```

File: docs/getting-started.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ We'll be going over the following steps:
 5. [Pushing new changes to the component](#5-pushing-new-changes-to-the-component)
 6. [Adding storage to the component](#6-adding-storage-to-the-component)
 
-**Extra documentation:**
+**Command completion**
+
+- [Command completion](#Command-completion)
```
Comment (mik-dass) [inline docs/getting-started.md]: The clickable link doesn't seem to work for me

Comment (amitkrout) [reply docs/getting-started.md]: @mik-dass Would you please let me know how you test it. However on mu macOS it works as expected. I use AsciiDoc in VS code editor to verify this.

Comment (geoand) [reply docs/getting-started.md]: It _did_ work for me as well

Comment (mik-dass) [reply docs/getting-started.md]: @amitkrout I viewed the file with "Display the rich diff" enabled on Github

File: docs/getting-started.md [hunk 2]
```diff
@@ -13,7 +13,11 @@ We'll be going over the following steps:
 5. [Pushing new changes to the component](#5-pushing-new-changes-to-the-component)
 6. [Adding storage to the component](#6-adding-storage-to-the-component)
 
-**Extra documentation:**
+**Command completion**
```
Comment (metacosm) [inline docs/getting-started.md]: I would just put `command completion` under `Extra documentation`. No need for an additional section, imo.

File: docs/getting-started.md (hunk 1 above)
Comment (syamgk) [reply docs/getting-started.md]: this works!
 but as @metacosm said this can be under extra docs

File: docs/getting-started.md (hunk 2 above)
Comment (amitkrout) [reply docs/getting-started.md]: Sounds good to me

Comment (amitkrout) [reply docs/getting-started.md]: Done

File: docs/getting-started.md (hunk 1 above)
Comment (mik-dass) [reply docs/getting-started.md]: @amitkrout Got it rich display doesn't work, I needed to do view file. Marking as resolved

Comment (amitkrout) [reply docs/getting-started.md]: Awesome!!! I have been using ```AsciiDoc``` since a long time without any issue. You can try it out ;)

**Pull request merged**